libraries/libfc/src/crypto/bls_signature.cpp

```diff
@@ -68,7 +31,7 @@ namespace fc { namespace crypto { namespace blslib {
 
       std::string data_str = fc::to_base58(v2, yield);
 
-      return std::string(config::bls_signature_base_prefix) + "_" + std::string(config::bls_signature_prefix) + "_" + data_str;
+      return data_str;

```

Comment:
I think we would want to keep a prefix for signatures. @arhag ?

Reply:
I don't even think we really need a string encoding for signatures. But if we did, then yes we would want the prefix. See https://github.com/AntelopeIO/leap/pull/1538#discussion_r1301981962.